modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -7,10 +7,17 @@
 
 When you create a VolumeSnapshot object, {product-title} creates a volume snapshot.
 
+
 .Prerequisites
 * Logged in to a running {product-title} cluster.
 * A PVC created using a CSI driver that supports VolumeSnapshot objects.
 * A storage class to provision the storage backend.
+* The Pod is not active.
```

Comment:
Here you want to say, no Pods are consuming the PVC which will be taken snapshot?

Reply:
> Here you want to say, no Pods are consuming the PVC which will be taken snapshot?

Yes, updated wording.